api/app.go

```diff
@@ -81,12 +81,34 @@ func (a *App) getRouter() *mux.Router {
 		&VersionMiddleware{},
 	)).Methods("GET").Name("oauth")
 
-	r.Handle("/clusters/{name}/create", Chain(
-		&ClusterHandler{App: a, Method: "create"},
+	r.Handle("/clusters/{name}/run", Chain(
+		&ClusterHandler{App: a, Method: "run"},
+		&LoggingMiddleware{App: a},
+		&VersionMiddleware{},
+		&AccessMiddleware{App: a},
+	)).Methods("PUT").Name("cluster")
```

Comment:
cluster table has a unique constraint on name. So creating the a config with an existing name doesn't insert new row on the table. This idempotent characteristic is better represented by a PUT, right?